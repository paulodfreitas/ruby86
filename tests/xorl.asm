.module entry_code
.pseg
.org 0
        irmovl $3, %edx
        irmovl $c, %ebx
        xorl %edx, %ebx
        halt
.end
