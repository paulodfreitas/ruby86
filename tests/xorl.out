@ 0000
30
82
03
00
00
00
30
83
0c
00
00
00
63
23
10
@ 000f
